galsim/table.py

```diff
@@ -165,18 +179,22 @@ def __call__(self, x):
             if dimen > 2:
                 raise ValueError("Arrays with dimension larger than 2 not allowed!")
             elif dimen == 2:
-                f = np.zeros_like(x)
-                for i in xrange(x.shape[0]):
-                    f[i,:] = np.fromiter((self.table(float(q)) for q in x[i,:]), dtype='float')
+                f = np.empty_like(x.flatten(), dtype=float)
```

Comment:
Yes.  Like I said, I had been using flatten, thinking it did what ravel does.  I changed all the placed in the galsim code from flatten -> ravel, since none of them seem to need a copy.  I didn't change anything in tests though.  All tests still pass.